databuilder/publisher/elasticsearch_constants.py

```diff
@@ -0,0 +1,149 @@
+import textwrap
+
+# Documentation: https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html
+# Setting type to "text" for all fields that would be used in search
+# Using Simple Analyzer to convert all text into search terms
+# https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-simple-analyzer.html
+# Standard Analyzer is used for all text fields that don't explicitly specify an analyzer
+# https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-standard-analyzer.html
+# TODO use amundsencommon for this when this project is updated to py3
+TABLE_ELASTICSEARCH_INDEX_MAPPING = textwrap.dedent(
```

Comment:
how about include the user index map as well?

Reply:
Good idea and done.